test/auth_test.rb

```diff
@@ -338,6 +338,19 @@ def app
       post "/oauth/token"
       assert_equal 422, last_response.status
     end
+
+    it "forwards a 403" do
+      stub_heroku_api do
+        post("/oauth/tokens") {
+          raise Excon::Errors::Forbidden.new("suspended", nil,
+            Excon::Response.new(body:
+              MultiJson.encode({id:"suspended", message:"ruh roh"}), status: 403))
```

Comment:
Line is too long. [82/80]<br>Redundant curly braces around a hash parameter.<br>Space missing after colon.<br>Space inside { missing.<br>Space inside } missing.